tests/EmitTest.elm

```diff
@@ -16,6 +17,37 @@ import Stage.Emit.JavaScript as JS
 import Test exposing (Test, describe, test, todo)
 
 
+typedHelp : Typed.Expr -> Typed.LocatedExpr
+typedHelp expr =
+    Located.located
+        -- position do not matters in emit
+        { start = { row = 0, col = 0 }, end = { row = 0, col = 0 } }
+        expr
+
+
+typed : Typed.Expr_ -> Typed.LocatedExpr
+typed expr =
+    Located.located
+        -- position do not matters in emit
```

Comment:
I think we can remove both of those comments, and extract that position to some top-level `dummyPosition = { ... }`